packages/server-setup/tasks-handlers.ts

```diff
@@ -46,14 +52,16 @@ const handlers = {
     );
   },
 
-  "upload-terminology": async (server: ConfigServer, task: ConfigTaskEntry) => {
+  "upload-terminology": async ({ server, task, plugin }: Handler) => {
     console.log(
```

Comment:
This is ok, but I'm sorry we didn't understand each other better, I didn't want you to remove the new handler type but to take care of the plugin inside of the plugin handler instead of in the general `handleTask`.

You don't have to revert back unless you feel like it.